lib/index.js

```diff
@@ -40,4 +40,4 @@
  *   console.log(result); // geojson FeatureCollection
  * });
  */
-module.exports = require('./binding/vtquery.node').vtquery;
+module.exports = require('./binding/module.node').vtquery;
```

Comment:
Is there a reason for this change?

Reply:
@axrj - looks like a mistake, let me check and correct.

Reply:
I made that, although not sure was the original issue just Travis issues (which there was yesterday), and I wouldn't have needed to do that.